### ISSUE TITLE
Failing to build on linux

### ISSUE THREAD

**Issue by lukassup** [rwengine/openrw#741]
```
[ 20%] Building CXX object rwengine/CMakeFiles/rwengine.dir/src/ai/TrafficDirector.cpp.o
In file included from /home/lukas/.cache/yay/openrw-git/src/openrw-git/rwengine/src/ai/TrafficDirector.cpp:1:
/home/lukas/.cache/yay/openrw-git/src/openrw-git/rwengine/src/ai/TrafficDirector.hpp:45:5: error: ‘size_t’ does not name a type
   45 |     size_t maximumPedestrians = 20;
      |     ^~~~~~
/home/lukas/.cache/yay/openrw-git/src/openrw-git/rwengine/src/ai/TrafficDirector.hpp:5:1: note: ‘size_t’ is defined in header ‘<cstddef>’; did you forget to ‘#include <cstddef>’?
    4 | #include <vector>
  +++ |+#include <cstddef>
    5 | 
/home/lukas/.cache/yay/openrw-git/src/openrw-git/rwengine/src/ai/TrafficDirector.hpp:46:5: error: ‘size_t’ does not name a type
   46 |     size_t maximumCars = 10;
      |     ^~~~~~
/home/lukas/.cache/yay/openrw-git/src/openrw-git/rwengine/src/ai/TrafficDirector.hpp:46:5: note: ‘size_t’ is defined in header ‘<cstddef>’; did you forget to ‘#include <cstddef>’?
/home/lukas/.cache/yay/openrw-git/src/openrw-git/rwengine/src/ai/TrafficDirector.cpp: In member function ‘std::vector<GameObject*> ai::TrafficDirector::populateNearby(const ViewCamera&, float, int)’:
/home/lukas/.cache/yay/openrw-git/src/openrw-git/rwengine/src/ai/TrafficDirector.cpp:154:9: error: ‘maximumPedestrians’ was not declared in this scope
  154 |     if (maximumPedestrians > world->pedestrianPool.objects.size()) {
      |         ^~~~~~~~~~~~~~~~~~
/home/lukas/.cache/yay/openrw-git/src/openrw-git/rwengine/src/ai/TrafficDirector.cpp:186:9: error: ‘maximumCars’ was not declared in this scope
  186 |     if (maximumCars > world->vehiclePool.objects.size()) {
      |         ^~~~~~~~~~~
/home/lukas/.cache/yay/openrw-git/src/openrw-git/rwengine/src/ai/TrafficDirector.cpp: In member function ‘void ai::TrafficDirector::setPopulationLimits(int, int)’:
/home/lukas/.cache/yay/openrw-git/src/openrw-git/rwengine/src/ai/TrafficDirector.cpp:267:5: error: ‘maximumPedestrians’ was not declared in this scope
  267 |     maximumPedestrians = maxPeds;
      |     ^~~~~~~~~~~~~~~~~~
/home/lukas/.cache/yay/openrw-git/src/openrw-git/rwengine/src/ai/TrafficDirector.cpp:268:5: error: ‘maximumCars’ was not declared in this scope; did you mean ‘maxCars’?
  268 |     maximumCars = maxCars;
      |     ^~~~~~~~~~~
      |     maxCars
make[2]: *** [rwengine/CMakeFiles/rwengine.dir/build.make:146: rwengine/CMakeFiles/rwengine.dir/src/ai/TrafficDirector.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:248: rwengine/CMakeFiles/rwengine.dir/all] Error 2
make: *** [Makefile:156: all] Error 2
```

Info

```
% uname -a
Linux marble 5.14.5-arch1-1 #1 SMP PREEMPT Thu, 16 Sep 2021 11:02:40 +0000 x86_64 GNU/Linux

% gcc --version
gcc (GCC) 11.1.0
...

% make --version
GNU Make 4.3
Built for x86_64-pc-linux-gnu

% cmake --version
cmake version 3.21.2
```

**Comment by lukassup** [comment]
Fix provided in #740

**Comment by tsjost** [comment]
Thanks, fix merged!
